.github/workflows/actions_build.yml

```diff
@@ -18,6 +18,7 @@ concurrency:
 env:
   LC_ALL: "en_US.UTF-8"
   BUILD_JDK_VERSION: "19"
+  GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
```

Comment:
JFYI, we can set an environment variable on GitHub settings. https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-an-environment